.github/workflows/docs.yml

```diff
@@ -11,11 +11,11 @@ on:
     types:
       - release-docs
   # Allow manual trigger for now
-  workflow_dispatch:
+  # workflow_dispatch:
```

Comment:
Any reason for removing this? It's handy for running ad-hoc manual releases

Reply:
It's not needed anymore I believe with the change you made. It now triggers after every PR with main and it will trigger with every release.